pkgs/tools/z/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, lib, ... }:
+
+stdenv.mkDerivation rec {
+  name = "z-${version}";
+  version = "1.11";
+
+  src = fetchFromGitHub {
+    owner = "rupa";
+    repo = "z";
+    rev = "v${version}";
+    sha256 = "13zbgkj6y0qhvn5jpkrqbd4jjxjr789k228iwma5hjfh1nx7ghyb";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  installPhase = ''
+    install -Dm 644 z.sh $out/share/z.sh
```

Comment:
Would it be possible to enable it in bash at install time? Becuase it seems you still need to do manual actions after this package installation to use it.

Reply:
(maybe the autojump package is doing this)

Reply:
What about zsh?

Reply:
maybe it is possible to do it for both shells

Reply:
I can't see if the autojump package is doing this, but I know that grml-zsh-config isn't. Not sure it's possible.

Reply:
In fact, a common way is to provide a script that returns the path of the script (since the path of the script in the nix store is not static).
For instance, `fzf` provides a script `fzf-share` that can be sourced in your bashrc file `source $(fzf-share)/completion.bash`.
See https://github.com/NixOS/nixpkgs/blob/ae702c68ccc5a8adf3178c880cfba9afcfcbf1f3/pkgs/tools/misc/fzf/default.nix#L44